R/timeseries.R

```diff
@@ -92,12 +92,13 @@ TimeSeries <- function(x = NULL,
         else
             stop("Row names of input data could not be interpreted as dates.")
     }
-    is.time <- !all(format(row.names, format = "%H:%M:%S") == "00:00:00")
-    rownames(x) <- as.character(row.names)
+    is.time <- !all(format(row.names.date, format = "%H:%M:%S") == "00:00:00")
+    rownames(x) <- as.character(row.names.date)
```

Comment:
I think `rownames(x)` still has the dates with inconsistent formatting but you aren't getting errors because you use `row.names.date` instead of `rownames(x)`. Maybe we should just delete this line. Otherwise substitute `as.character` for `format`